### PR TITLE
Scope whitelisted global namespace classes

### LINKED ISSUE #10: Add namespace to non-namespaced classes
Currently the classes without namespace (for example https://github.com/sebastianbergmann/phpunit/blob/master/src/Framework/TestCase.php) are just ignored

### PR DESCRIPTION
**Pull request by theofidry**
Closes #10

### CODE REVIEW

File: src/NodeVisitor/GlobalWhitelistedNamesNodeVisitor.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function enterNode(Node $node)
+    {
+        if ($node instanceof Name
+            && 1 === count($node->parts)
+            && true === ($this->whitelister)($node->parts[0])
```
Comment (atimmer) [inline src/NodeVisitor/GlobalWhitelistedNamesNodeVisitor.php]: In the `Name` class PHPParser writes:

> Avoid directly accessing $parts, use methods instead.

So I think `$node->parts[0]` should become `$node->getFirst()`

**Comment by atimmer** [comment]
I'm testing this and it seems that it doesn't prefix the name in the `class SomeClass` definition. It looks like the name of a `Class_` isn't actually a `Name` object, but just a string.

**Comment by theofidry** [comment]
Thanks for testing :) Could you provide a more complete example?

File: src/NodeVisitor/GlobalWhitelistedNamesNodeVisitor.php (hunk 1 above)
Comment (padraic) [inline src/NodeVisitor/GlobalWhitelistedNamesNodeVisitor.php]: @atimmer I cam across this before, but the current status is that there is no public API for parts. Once it's added, we can move across. The deprecation seems more of a TODO in this case.

**Comment by theofidry** [comment]
PR updated, ready for review

**Pull request merged**